pkg/endpoint/endpoint.go

```diff
@@ -945,7 +955,7 @@ type DeleteConfig struct {
 // endpoints which failed to be restored. Any cleanup routine of leaveLocked()
 // which depends on kvstore connectivity must be protected by a flag in
 // DeleteConfig and the restore logic must opt-out of it.
-func (e *Endpoint) leaveLocked(proxyWaitGroup *completion.WaitGroup, conf DeleteConfig) []error {
+func (e *Endpoint) leaveLocked(proxyWaitGroup *completion.WaitGroup, allocator cache.IdentityAllocator, conf DeleteConfig) []error {
```

Comment:
This seems to be a leftover since we have the allocator present in the endpoint.